x-pack/plugins/data_enhanced/common/search/types.ts

```diff
@@ -15,20 +15,6 @@ import {
 
 export const ENHANCED_ES_SEARCH_STRATEGY = 'ese';
 
-export interface IAsyncSearchRequest extends IEsSearchRequest {
```

Comment:
No need for this type since the base `IKibanaSearchRequest` already has an `id`.